.buildkite/bench_pipeline.yml

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+steps:
```

Comment:
Looks like this can be now updated.

Reply:
Not yet. If I update the pipeline with this PR, the CI will try to run the benchmarks on upstream/master, where they're obviously not merged yet => CI will fail. This PR needs to go in first, then the pipeline will be updated in a new one.

Reply:
This is no longer the case. I've updated the test_bench such that when master has no performance tests, it just runs the test on the PR. PR for reference: https://github.com/rust-vmm/rust-vmm-ci/pull/31. We modified this behaviour while you were on vacation :)) thought it's a bug.